package.json

```diff
@@ -3,13 +3,17 @@
   "description": "JavaScript implementation of the \"accept\" attribute for HTML5 <input type=\"file\">",
   "version": "0.0.0-development",
   "main": "dist/index.js",
-  "module": "dist/es/index.js",
+  "module": "src/index.js",
+  "exports": {
+    "import": "./dist/index.mjs",
```

Comment:
You should provide types in the export map: https://publint.dev/rules#types_not_exported

Reply:
Added! Sorry for the delay.